scripts/test-examples

```diff
@@ -61,11 +61,16 @@ function bootBroccoli() {
       examplesEnv.EXAMPLES = 1;
 
       var broccoli = childProcess.exec('node ' + broccoliBin + ' serve', { env: examplesEnv });
+      var broccoliOutput = "";
 
       broccoli.stdout.on('data', function (data) {
-        log('broccoli', data);
```

Comment:
output doesn't necessarily end either. The approach of adding to the buffer is great, but I wouldn't assume we get an end event.